src/components/banner/Banner.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Icon from '../icon';
+import { BannerType, BannerIcon } from './BannerUtils';
+
+type BannerProps = {
+  /** Optional CSS class name */
+  className?: string;
+  /** If true, close icon will be placed to the right */
+  canClose?: boolean;
+  /** Function to call on close action */
+  onClickClose?: () => void;
+  /** Content of the Banner */
+  content: string;
+  /** If Banner should be shown or not */
+  show?: boolean;
+  /** Style of the banner */
+  variant?: BannerType;
+  /** Text of the action button */
+  actionText?: string;
+  /** Function to call on action click */
+  onClickAction?: () => void;
+  /** Size of the banner */
+  size?: 'small' | 'medium';
+
+  /** other props */
+  [otherProps: string]: any,
+}
+
+export const Banner: React.FC<BannerProps> = ({
+  className,
+  canClose,
+  onClickClose,
+  actionText,
+  onClickAction,
+  content,
+  show,
+  variant,
+  size,
+  ...otherProps
+}) => {
+
+  if (!show) {
+    return null
+  }
+
+  return (
+    <div className={classNames(
+      'tk-banner',
+      `tk-banner__${variant}`,
```

Comment:
make sure that variant exists, otherwise you will have a class "tk-banner__"

Reply:
Yes, there is a default value.
```
Banner.defaultProps = {
  variant: BannerType.NEUTRAL,
  ...
}
```

Reply:
Ok sounds good! I was afraid of the optional prop but ok you handle it